generators/entity-server/index.js

```diff
@@ -24,14 +24,9 @@ const writeFiles = require('./files').writeFiles;
 module.exports = class extends EntityServerGenerator {
     constructor(args, opts) {
         super(args, { fromBlueprint: true, ...opts }); // fromBlueprint variable is important
-
-        const jhContext = (this.jhipsterContext = this.options.jhipsterContext || opts.jhipsterContext);
```

Comment:
I think `this.jhipsterContext` assignment is valid and we need to have it.

Reply:
According to https://github.com/jhipster/generator-jhipster/pull/9942 it's done by parent generator, so you don't need to do it in blueprint generator

Reply:
Interesting, let me have a look. Sorry for taking too long to reply. Catching up things on the opensource side.

Reply:
That is interesting.